src/components/pages/Home/MainHeader.js

```diff
@@ -75,7 +75,9 @@ export default function MainHeader(props) {
             <div className="user-button">
               <Icon
                 component={() => (
-                  <img id="avatar-icon" src={Profile} alt="profile icon" />
+                  <Link to="/account">
```

Comment:
This is going to the wrong spot, /account should be just /

Reply:
I believe linking it to ' / ' would bring the user back to the home page, what we want is when the user clicks the avatar icon, they'll be brought to their account page.